gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/types/card-options.functions.ts

```diff
@@ -0,0 +1,14 @@
+const foilSuffix = '*';
```

Comment:
in a perfect world I'd pull these constants into their own file so they're easy to find/reuse.

Reply:
oh! i did, they're in jCard.constants.ts.

Nuke these and use those